.browserslistrc

```diff
@@ -0,0 +1,5 @@
+# Browsers that we support
```

Comment:
Hm, you are right... I think I would simply remove that comment, since it doesn't add any value.